designing-event-driven-processes/README.md

```diff
@@ -0,0 +1,18 @@
+# Beyond basic messaging: Designing long-running processes
+
+## Abstract
+
+Messaging in distributed systems offers a range of valuable benefits: it enables the design of decoupled components, where each component can evolve and scale individually. In addition, messaging improves the system’s fault tolerance but goes even further by opening the door to building self-healing systems. One benefit that doesn’t usually appear in the Googleable list of benefits is one of the most powerful: supporting real-life, long-running business processes and making the system mirror real-world business use cases. And that’s exactly the focus of this workshop.
+
+If you’ve been exposed to the basic building blocks of messaging and want to understand how to design, build, and test long-running processes in message-based distributed systems, this workshop is for you.
```

Comment:
This sentence and the title assume the audience knows what "long-running (business) processes" are. In my experience, that is not the case. "long-running" is particularly tricky as it implies a reference to time, and it implies it takes a lot of time. In reality, we know that whatever is "longer" than the span of a single transaction is a good candidate for a saga/workflow/long-running process.

I don't know how to rephrase it, though.

Reply:
That's one of the first topics I'm planning to start the workshop with. Do you think it's important enough to clarify this in the abstract, as in, because they don't really know what long-running is they won't think this is a good fit for them?